### ISSUE TITLE
Different ways of Encoding Categorical Data in Data pre-processing steps of Machine Learning

### ISSUE THREAD

**Issue by chandu6111** [prathimacode-hub/DS-ScriptsNook#10]
---
Title: Different ways of Encoding Categorical Data in Data pre-processing steps of Machine Learning
Name: Chandana (chandu6111)
Label: Feature Request

---

Define You:

- [x] DevIncept Participant 
- [x] Contributor

**Is your feature request related to a problem? Please describe.**
They are quite a good number of types of Data Encoding. Encoding refers to converting Categorical Data into numerical data. This is an essential step in Data Preprocessing.

So, I would like to explain different types of data encoding by taking a sample dataset as an example. This would come under Machine Learning folder but not under Algorithms, it is a data preprocessing step.

**Comment by chandu6111** [comment]
@prathimacode-hub I would like to contribute to this under DCP`21
could you please assign me this issue? Or if there is anything specific you would like to know about the issue?

**Comment by prathimacode-hub** [comment]
That's great to know. Yes follow the project structure and you are good to go. I had created a "Tutorials" folder in Machine Learning where you could add your repo. 👍@chandu6111

**Comment by prathimacode-hub** [comment]
Also can you make the issue title short stating specifically on the topic. @chandu6111

**Comment by prathimacode-hub** [comment]
Your PR hasn't turned up yet. It's been long since created up the issue. @chandu6111

**Comment by chandu6111** [comment]
Hey @prathimacode-hub , due to me being busy with exams, I couldn't make a PR yet. I will make it in a week, if you could give me some time, it would be great 👍

**Comment by prathimacode-hub** [comment]
Sure, you can. @chandu6111

**Comment by prathimacode-hub** [comment]
Yup, any update on this issue? Hope you know that last day is arriving up for DCP program. @chandu6111

**Comment by prathimacode-hub** [comment]
Its been 5 days. 9th is the last date to accept PR's. If you wouldn't do it by then, obviously you can submit later but points won't be counted towards DCP program. @chandu6111

**Comment by chandu6111** [comment]
Okay fine @prathimacode-hub 
My exams schedule is very tight actually.. otherwise I would have submitted it already. I will contribute to this issue after the DCP program also... you can remove DCP label if you want to..

**Comment by prathimacode-hub** [comment]
> Okay fine @prathimacode-hub
> My exams schedule is very tight actually.. otherwise I would have submitted it already. I will contribute to this issue after the DCP program also... you can remove DCP label if you want to..

Ok 👍

**Comment by prathimacode-hub** [comment]
Update? @chandu6111

**Comment by prathimacode-hub** [comment]
Update please? @chandu6111

**Comment by chandu6111** [comment]
Sorry @prathimacode-hub , I missed on your previous notification. You could make it up-for-grab, because I am somewhat busy with all placement exams. Sorry for the inconvenience caused.

**Comment by prathimacode-hub** [comment]
No worries. All the best for your exams. 😎👍@chandu6111

**Comment by deepthi1107** [comment]
please do assign me @prathimacode-hub

**Comment by prathimacode-hub** [comment]
Yes, I can assign but since this issue is created by some one else. Add up your approach and as part of what open source program are you contributing? @deepthi1107

**Comment by deepthi1107** [comment]
mam @prathimacode-hub I have made changes in my comment mam.

**Comment by prathimacode-hub** [comment]
Ok, issue is assigned to @deepthi1107
